src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day">
+      <div class="calendar__day"></div>
```

Comment:
The calendar should have a month-length modifier, like 'calendar--month-length_28'. Use one of the values: 28, 29, 30, or 31.

Reply:
Not resolved. If you have any questions - feel free to ask them in `fe_chat`